src/services/cronJobs/syncUserPowers.ts

```diff
@@ -129,7 +142,18 @@ export function processSyncUserPowerJobs() {
           givbackRound,
           users,
         });
-        logger.debug('inserting userPowers...', users.length);
+        const remainingJobsCount = await syncUserPowersQueue.count();
+        if (remainingJobsCount === 0) {
+          await setPowerRound(givbackRound);
```

Comment:
@aminlatifi  I think it's the best place for updating current round because all user powers are synced and we calculated round well, 
what do you think?